source/assets/stylesheets/styles/_footer.scss

```diff
@@ -0,0 +1,159 @@
+.footer {
+  background-color: $dark-gray;
+  min-height: 280px;
+}
+
+.footer-container {
+  @extend %container;
+  box-sizing: border-box;
+  color: $white;
+  font-size: 0.85em;
+  padding: 30px 16px 50px;
+
+  @include breakpoint($tablet-breakpoint) {
+    padding: 50px 0;
+  }
+}
+
+.footer-container__address-tile {
+  border-bottom: groove 1px;
+  box-sizing: border-box;
+  display: block;
+  font-weight: 100;
+  line-height: 1.5em;
+  padding-bottom: 12px;
+  padding-right: 16px;
+  width: 100%;
+
+  @include breakpoint($tablet-breakpoint) {
+    float: left;
+    border-bottom: none;
+    padding-bottom: 40px;
+    padding-left: 16px;
+    width: 33.33%;
+  }
+}
+
+.footer-container__address-city {
+  font-weight: 400;
+  padding: 15px 0 10px;
+
+  a {
+    color: inherit;
+    text-decoration: none;
+  }
+
+  &:after {
+    content: ">";
+    padding-left: 8px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    padding-bottom: 25px;
+    padding-top: 0;
+
+    span {
+      display: none;
+    }
+
+    &:after {
+      content: ""
+    }
+  }
+}
+
+.footer-container__address-phone {
+  font-weight: 400;
+  color: inherit;
+  text-decoration: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    pointer-events: none;
+    cursor: text;
+  }
+}
+
+.footer-container__address-full {
+  display: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    display: block;
+  }
+}
+
+.footer-container__registration-address {
```

Comment:
I'm guessing that this should be displayed somewhere, including the SA registration details..?